core/src/test/scala/org/apache/spark/sql/NorthWindTest.scala

```diff
@@ -0,0 +1,1481 @@
+/*
+ * Copyright (c) 2016 SnappyData, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+package org.apache.spark.sql
+
+import io.snappydata.SnappyFunSuite
+import org.apache.spark.Logging
+import org.apache.spark.sql.execution.aggregate.{SortBasedAggregate, TungstenAggregate}
+import org.apache.spark.sql.execution.joins._
+import org.apache.spark.sql.execution.{LocalTableScan, PartitionedPhysicalRDD, Project, Sort}
+import org.scalatest.{BeforeAndAfter, BeforeAndAfterAll}
+
+class NorthWindTest
```

Comment:
NorthWind Trader's is fictitious company name, not any real SnappyData customer's name.